src/commands/projects/list.ts

```diff
@@ -58,30 +95,64 @@ Use this command to list projects that are part of a particular organization.`
 
   async buildRequestParameters(options: Partial<flags.Output>): Promise<ProjectsListParams> {
     debug('buildRequestParameters()')
-    const {organization: orgId} = options
+    const {
+      'exclude-channels': excludeChannels,
+      'include-features': includeFeatures,
+      limit = defaultLimit,
+      offset,
+      organization: orgId,
+      sort: sortBy,
+      'with-name': filter,
+    } = options
 
-    return {orgId}
+    return {
+      excludeChannels,
+      includeFeatures,
+      ...(typeof filter === 'undefined' ? {} : {filter}),
+      ...(typeof limit === 'undefined' ? {} : {limit}),
+      ...(typeof offset === 'undefined' ? {} : {offset}),
+      ...(typeof orgId === 'undefined' ? {} : {orgId}),
+      ...(typeof sortBy === 'undefined' ? {} : {sortBy}),
+    }
   }
 
   doRequest(client: MixClient, params: ProjectsListParams): Promise<MixResponse> {
     debug('doRequest()')
     return ProjectsAPI.listProjects(client, params)
   }
 
-  outputHumanReadable(transformedData: any) {
+  outputHumanReadable(transformedData: any, options: any) {
     debug('outputHumanReadable()')
+    let isExcludeChannel = false
+    let isIncludeFeatures = false
+
     if (transformedData.length === 0) {
       this.log('No projects found.')
 
       return
     }
 
-    this.outputCLITable(transformedData, this.columns)
+    options['exclude-channels'] ? isExcludeChannel = true : isExcludeChannel = false
+    options['include-features'] ? isIncludeFeatures = true : isIncludeFeatures = false
+
+    let tableColumns = {}
+
+    isIncludeFeatures ? (isExcludeChannel ? tableColumns = {...this.columns, ...this.featuresColumn, ...this.timeColumns} :
+      tableColumns = {...this.columns, ...this.featuresColumn, ...this.channelsColumn, ...this.timeColumns}) :
+      (isExcludeChannel ? tableColumns = {...this.columns, ...this.timeColumns} :
+        tableColumns = {...this.columns, ...this.channelsColumn, ...this.timeColumns})
+
+    this.outputCLITable(transformedData, tableColumns)
+
```

Comment:
We are going to need to output something like "Showing results 1-25 of 200." at the end of the list. If the list contains a single project then it should say "Showing result 1 of 1."